PurrSoft/PurrSoft.Api/Controllers/AnimalFosterMapController.cs

```diff
@@ -0,0 +1,104 @@
+﻿using FluentValidation;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using PurrSoft.Api.Controllers.Base;
+using PurrSoft.Application.Common;
+using PurrSoft.Application.Models;
+using PurrSoft.Application.Queries.AnimalFosterMapQueries;
+using PurrSoft.Application.QueryOverviews;
+using System.Net;
+using static PurrSoft.Application.Commands.AnimalFosterMapCommands.AnimalFosterMapCommands;
+
+namespace PurrSoft.Api.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class AnimalFosterMapController : BaseController
+{
+	[HttpGet()]
+	[Authorize(AuthenticationSchemes = "Bearer", Roles = "Manager, Foster")]
+	[ProducesResponseType(typeof(CollectionResponse<AnimalFosterMapDto>), (int)HttpStatusCode.OK)]
```

Comment:
Your response type does not match your return type, it says that it should response with a CollectionResponse of AnimalFosterMapDto, but it returns a CollectionResponse of AnimalFosterMapOverview.